src/NRules/NRules.Testing/IRuleFireAssertion.cs

```diff
@@ -0,0 +1,10 @@
+﻿using NRules.Fluent.Dsl;
+
+namespace NRules.Testing;
+
+public interface IRuleFireAssertion
+{
+    IRuleFireAssertResult IsFiredTimes(int expected);
+
+    IRuleFireAssertResult IsFiredTimes<T>(int expected) where T : Rule;
```

Comment:
@gritcsenko probably better to say HasFiredTimes from the English readability perspective

Reply:
This interface was leftover. Removed